### PR TITLE
added Intellij Idea Ultimate and Goland under Linux IDE options

### LINKED ISSUE #978: IntelliJ Idea Ultimate not supported by editor integration
### The problem

I am unable to see my IDE in the options.
Under Linux. Using InteliJ Idea Ultimate

### Release version

3.3.5

### Operating system

Ubuntu 22.04

### Steps to reproduce the behavior

go to file
options
integrations

image

### Log files

_No response_

### Screenshots

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Madeindreams**
Closes #978

## Description

- Added intellij idea ultimate and goland under linux snap installation folder

### Screenshots

image
## Release notes

Notes:

### CODE REVIEW

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution @Madeindreams!

**Comment by Madeindreams** [comment]
@shiftkey Note that I have also updated the documentation after the approval!

**Comment by shiftkey** [comment]
@Madeindreams looks like there's a linter issue reported - if you run `yarn lint:fix` locally that should give you the changes you need to commit

File: docs/technical/editor-integration.md [hunk 1]
```diff
@@ -353,6 +353,8 @@ These editors are currently supported:
  - [Lite XL](https://lite-xl.com/)
  - [JetBrains PHPStorm](https://www.jetbrains.com/phpstorm/)
  - [JetBrains WebStorm](https://www.jetbrains.com/webstorm/)
+ - [JetBrains Idea Ultimate](https://www.jetbrains.com/idea/)
+ - [JetBRains Goland](https://www.jetbrains.com/go/)
```
Comment (shiftkey) [inline docs/technical/editor-integration.md]:
```suggestion
 - [JetBrains Goland](https://www.jetbrains.com/go/)
```

Comment (Madeindreams) [reply docs/technical/editor-integration.md]: Ha good catch!

**Comment by Madeindreams** [comment]
What else is required to merge this PR?

**Comment by shiftkey** [comment]
@Madeindreams I'm working on getting the fork updated to the latest upstream code, which requires detangling some dependency updates. I'll handle merging this in when it's ready and then adding it to the next release.

**Pull request closed without merging**